### PR TITLE
Embree & OpenPGL fix supported arches and REQUIRED_USE

### PR DESCRIPTION
**Pull request by DarkDefender**
Both Embree and OpenPGL only support and compiles on amd64 and arm64.

Embree needs a small patch to compile on arm64 on Linux

### CODE REVIEW

File: media-libs/embree/embree-3.13.5.ebuild [hunk 1]
```diff
@@ -11,13 +11,11 @@ SRC_URI="https://github.com/embree/embree/archive/v${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="Apache-2.0"
 SLOT="3"
-KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
-X86_CPU_FLAGS=( sse2:sse2 sse4_2:sse4_2 avx:avx avx2:avx2 avx512dq:avx512dq )
+KEYWORDS="~amd64 ~arm64"
```
Comment (thesamesam) [inline media-libs/embree/embree-3.13.5.ebuild:14]: If it definitely doesn't build on other arches, let's -* it too

File: media-libs/embree/files/embree-3.13.5-fix-arm64.patch [hunk 2]
```diff
@@ -0,0 +1,48 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
```
Comment (thesamesam) [inline media-libs/embree/files/embree-3.13.5-fix-arm64.patch:1]: Link to the commit source + any relevant gentoo bugs too, pls (at the top of the patch)

**Comment by DarkDefender** [comment]
Hmm, I'm unsure how to solve the issues in the CI report.
Both embree and openpgl are optional deps for Blender.
So one could still, at least in theory, build it for the `arm` arch.

**Comment by thesamesam** [comment]
package.use.mask in arch/arm

**Pull request closed without merging**